sass/components/_carousel.scss

```diff
@@ -3,12 +3,21 @@
 }
 
 /* Slideshow container */
-.slideshow-container {
-  height: 32vh;
-  max-width: 1000px;
-  position: relative;
-  margin: auto;
-  z-index: -10;
+.carousel {
+  &-container {
```

Comment:
I recommend avoiding using such selectors `&-container`, etc.. You might be reusing parts of a selector, but you're making it harder to debug when you want to search for `carousel-container`. There's nothing wrong with having explicit code.

```css
.carousel-container {
  ...
}

.carousel-image {
  ...
}
```